### PR TITLE
Add blogpost.

### PR DESCRIPTION
**Pull request by aressem**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -55,6 +55,7 @@ authors:
     display_name: Arnstein Ressem
     avatar: '/assets/avatars/aressem.jpg'
```
Comment (kkraune) [inline _config.yml:56]:
```suggestion
    avatar: '/assets/avatars/aressem.png'
```

File: _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "How Using Screwdriver for CI/CD Reduced Vespa’s Time Spent on Builds and Pull Requests by 75%"
+date: '2021-02-21'
```
Comment (kkraune) [inline _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md:4]:
```suggestion
date: '2021-02-22'
```
lets merge and publish on monday

File: _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+tags: []
+author: aressem
+image: assets/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent/cover.jpg
+excerpt: Introducing Screwdriver for Vespa's CI/CD needs.
```
Comment (kkraune) [inline _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md:8]:
```suggestion
excerpt: Introducing Screwdriver for Vespa's CI/CD needs.
skipimage: true
```
skip auto image inclusion, it it included below with credit

File: _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+*By [Arnstein Ressem](https://www.linkedin.com/in/arnsteinressem/), Principal Software Systems Engineer, Verizon Media*
+
+When Vespa was [open sourced in 2017](https://blog.vespa.ai/open-sourcing-vespa-yahoos-big-data-processing/) we looked for a continuous integration platform to build our source code on. We looked at several hosted solutions as well as [Screwdriver](https://screwdriver.cd/) – an open source CI/CD platform built by Yahoo/Verizon Media – that had just been open sourced in 2016. Another platform seemed the best fit for us at that point in time and we integrated with that.
+
+![Decorative image](/assets/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent/cover.jpg)
+<p class="image-credit">*Photo by <a href="https://unsplash.com/@bill_oxford?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Bill Oxford</a> on <a href="https://unsplash.com/s/photos/cogs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash*</a></p>
```
Comment (kkraune) [inline _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md:16]: are the * in image captions intentional or meant as markup?

**Review by kkraune — APPROVED** [review]
Great! lets merge on monday

File: _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md (hunk 4 above)
Comment (aressem) [reply _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md:16]: It was done to get the captions in italic as suggested in the reviews. Does this not work ?

File: _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+excerpt: Introducing Screwdriver for Vespa's CI/CD needs.
+skipimage: true
+
+---
+*By [Arnstein Ressem](https://www.linkedin.com/in/arnsteinressem/), Principal Software Systems Engineer, Verizon Media*
+
+When Vespa was [open sourced in 2017](https://blog.vespa.ai/open-sourcing-vespa-yahoos-big-data-processing/) we looked for a continuous integration platform to build our source code on. We looked at several hosted solutions as well as [Screwdriver](https://screwdriver.cd/) – an open source CI/CD platform built by Yahoo/Verizon Media – that had just been open sourced in 2016. Another platform seemed the best fit for us at that point in time and we integrated with that.
+
+![Decorative image](/assets/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent/cover.jpg)
+<p class="image-credit">*Photo by <a href="https://unsplash.com/@bill_oxford?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Bill Oxford</a> on <a href="https://unsplash.com/s/photos/cogs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash*</a></p>
```
Comment (kkraune) [inline _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md:17]:
```suggestion
<p class="image-credit"><em>Photo by <a href="https://unsplash.com/@bill_oxford?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Bill Oxford</a> on <a href="https://unsplash.com/s/photos/cogs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></em></p>
```

File: _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+In the fall of 2020 we became aware of big changes in the selected CI/CD platform and we needed to migrate to something else. As part of this work we took another look at the open sourced version of [Screwdriver](https://screwdriver.cd/) as we knew that the project had significantly matured over the past years. [Screwdriver](https://screwdriver.cd/) is an open source build platform designed for Continuous Delivery that can easily be deployed on different IaaS providers and is currently an incubee in the [Continuous Delivery Foundation](https://cd.foundation/).
+
+![Screwdriver](/assets/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent/screwdriver.png)
+<p class="image-credit">*Vespa pipeline on Screwdriver*</p>
```
Comment (kkraune) [inline _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md:26]:
```suggestion
<p class="image-credit"><em>Vespa pipeline on Screwdriver</em></p>
```

File: _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+The Vespa team got access to a hosted instance at [cd.screwdriver.cd](https://cd.screwdriver.cd) (*invite only, but publicly readable with guest access*). Working closely with [Screwdriver](https://screwdriver.cd/) we were able to reduce the build times for the master branch and pull requests from 50 minutes on the previous solution to 18 minutes. This result was obtained by using [Screwdriver](https://screwdriver.cd/)’s configurable resource management and fast build caches. We also appreciated the small set of requirements on container images allowing us to optimize the build image for our jobs.
+
+![Github](/assets/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent/github.png)
+<p class="image-credit">*Screwdriver integrated with pull request builds on GitHub*</p>
```
Comment (kkraune) [inline _posts/2021-02-19-how-using-screwdriver-for-cicd-reduced-vespas-time-spent-on-builds-and-pull-requests-by-75-percent.md:31]:
```suggestion
<p class="image-credit"><em>Screwdriver integrated with pull request builds on GitHub</em></p>
```

**Review by kkraune — APPROVED** [review]
italics

**Pull request merged**